### PR TITLE
PHOENIX-6530 - changed the tenant id generation for uniform distribution load generator

### PR DESCRIPTION
**Pull request by thrylokya**
changed UniformDistributionLoadEventGenerator. added numTenants attribute for generating the tenant id.

### CODE REVIEW

**Comment by mnpoonia** [comment]
Thanks for the contribution. @thrylokya 
@jpisaac FYI

File: phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/generators/UniformDistributionLoadEventGenerator.java [hunk 1]
```diff
@@ -72,12 +76,22 @@ public UniformDistributionSampler(List<Operation> operationList, DataModel model
         public TenantOperationInfo nextSample() {
             int sampleIndex = this.distribution.nextInt(operationList.size());
             Operation op = operationList.get(sampleIndex);
+            int numTenants = 1;
+
+            if(tenantGroup.getNumTenants() != 0)
+            {
```
Comment (gjacoby126) [inline phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/generators/UniformDistributionLoadEventGenerator.java:82]: tiny nit: angle bracket on the previous line

File: phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/generators/UniformDistributionLoadEventGenerator.java [hunk 2]
```diff
@@ -72,12 +76,22 @@ public UniformDistributionSampler(List<Operation> operationList, DataModel model
...
+                numTenants = tenantGroup.getNumTenants();
+            }
 
             String tenantGroupId = tenantGroup.getId();
             String tenantIdPrefix = Strings
                     .padStart(tenantGroupId, loadProfile.getGroupIdLength(), 'x');
+            // String formattedTenantId = String.format(loadProfile.getTenantIdFormat(),
```
Comment (gjacoby126) [inline phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/mt/generators/UniformDistributionLoadEventGenerator.java:89]: nit: no need to add commented out code

**Review by gjacoby126 — COMMENTED** [review]
Couple of nits. Also please change the PR and commit message to include a JIRA number

**Pull request merged**